swagger/public_v1/template.yml

```diff
@@ -435,6 +435,66 @@ components:
             $ref: "#/components/schemas/LocationResource"
         jsonapi:
           $ref: "#/components/schemas/JSONAPI"
+    LocationStatus:
+      type: object
+      description: Wrapper object for LocationStatusAttributes
+      properties:
+        id:
+          type: string
+          example: "22329867"
+        type:
+          type: string
+          example: site_statuses
+        attributes:
+          $ref: "#/components/schemas/LocationStatusAttributes"
+    LocationStatusAttributes:
```

Comment:
What about putting the attributes file into the `component_schemas` dir as a pattern? That's what we did with `CourseAttributes` and `ProviderAttributes`.

Reply:
That's not in master yet and only exists in http://github.com/DFE-Digital/teacher-training-api/pull/1278/

Reply:
That PR is merged now.

Reply:
i've rebased and split out so it aligns with the other files